frontend/prebuild/src/app/login/login.component.ts

```diff
@@ -22,26 +22,40 @@ export class LoginComponent implements OnInit {
     private router: Router,
     private ngZone: NgZone,
     private meta: Meta,
-    private http: HttpClient
+    private http: HttpClient,
+    private route: ActivatedRoute
   ) {}
 
+
   ngOnInit() {
+  
     this.meta.removeTag('viewport');
     let viewWidth = window.innerWidth;
     let viewHeight = window.innerHeight;
 
     this.meta.addTag({name: 'viewport', content: `width=${viewWidth}, height=${viewHeight}, initial-scale=1.0`}, true);
-
+    
+    this.route.params.subscribe( params => 
+    {
+      localStorage.setItem("jwt", params['jwt']);
+    });
+    if (localStorage.getItem('jwt') !== null && localStorage.getItem('jwt') !== 'undefined') {
+      this.loginThroughGoogle();
+    }
+    
     if (sessionStorage.getItem('username') !== null) {
       this.username = sessionStorage.getItem('username');
       this.player.name = this.username;
     }
   }
+  
+  loginGoogle() {
+       window.location.href = `http://${document.location.hostname}:8082/auth-service/GoogleAuth`;
```

Comment:
this won't work on bluemix because the auth-service won't have the same hostname as frontend.  Instead, you can add `API_URL_AUTH` to `frontend/prebuild/src/environments/environments.ts` and then reference it here (and I will override the environment value on bluemix)